app/views/admin/orders/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<h1 align = "center">Order Info for Order #<%= "#{@order.id}"%></h1>
+<center>
+  <table>
+    <tr>
+      <th>Item</th>
```

Comment:
Many thanks to who ever worked on the user order show page, most of this is copied from there.

Reply:
I think it may have come with the repo when we downloaded it, and then we just tweaked it to fit the specifications :)